update the tidb configuration documents. (#4056)
pingcap/docs: command-line-flags-for-tidb-configuration.md

- Default: ""
- If you have specified the configuration file, TiDB reads the configuration file. If the corresponding configuration also exists in the command line options, TiDB uses the configuration in the command line options to overwrite that in the configuration file. For detailed configuration information, see [TiDB Configuration File Description](/tidb-configuration-file.md).

## `--config-check`

> - check config file validity and exit

Reviewer: ```suggestion
- Checks the validity of the configuration file and exits
```

- Default: false

## `--config-strict`

> - enforce config file validity

Reviewer: ```suggestion
- Enforce the validity of the configuration file
```

> - Enforce the validity of the configuration file

Reviewer: ```suggestion
- Enforces the validity of the configuration file
```

- Default: "info"
- You can choose from "debug", "info", "warn", "error", or "fatal".

## `--lease`

> - schema lease duration, very dangerous to change only if you know what you do

Reviewer: ```suggestion
- The duration of the schema lease. It is very dangerous to change the value unless you know what you do.
```

> - The duration of the schema lease. It is very dangerous to change the value unless you know what you do.

Reviewer: ```suggestion
- The duration of the schema lease. It is **dangerous** to change the value unless you know what you do.
```

> - Default: "45s"

Reviewer: ```suggestion
- Default: 45s
```
